CMakeLists.txt

```diff
@@ -223,14 +225,15 @@ if (MI_BUILD_STATIC)
   message(STATUS "Static library will be built")
 
   add_library(mimalloc-static STATIC ${mi_sources})
+  set_property(TARGET mimalloc-static PROPERTY POSITION_INDEPENDENT_CODE ON)
```

Comment:
What is the rationale for this?

Reply:
That is the merge with PR #240.
The rationale is that some builds require linking with -fPIC code, and nowadays the performance degradation is negilible so we just always build PIC code.

Reply:
Well... certainly depends on the architecture.... but ok.